conf/ems/9.6.0/ems.yaml

```diff
@@ -4,232 +4,409 @@ object: ems
-      - parameters.volumeUuid           => volume_uuid
+      - parameters.op                       => op
+      - parameters.volumeUuid               => volume_uuid
+      - parameters.volumeName               => volume
+      - parameters.vserverName              => svm
+      - parameters.vserverUuid              => svm_uuid
 
   - name: arw.volume.state
     matches:
       - name: op
         value: dry-run-paused
     exports:
-      - parameters.op                   => op
-      - parameters.volumeUuid           => volume_uuid
+      - parameters.op                      => op
+      - parameters.volumeUuid              => volume_uuid
+      - parameters.volumeName              => volume
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
   - name: arw.volume.state
     matches:
       - name: op
         value: enable-paused
     exports:
-      - parameters.op                   => op
-      - parameters.volumeUuid           => volumeuuid
+      - parameters.op                      => op
+      - parameters.volumeUuid              => volume_uuid
+      - parameters.volumeName              => volume
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
   - name: arw.volume.state
     matches:
       - name: op
         value: enabled
     exports:
-      - parameters.op                   => op
-      - parameters.volumeUuid           => volume_uuid
+      - parameters.op                      => op
+      - parameters.volumeUuid              => volume_uuid
+      - parameters.volumeName              => volume
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
   - name: arw.vserver.state
     matches:
       - name: op
         value: enabled
     exports:
-      - parameters.op                   => op
-      - parameters.vserverName          => svm
-      - parameters.vserverUuid          => svm_uuid
+      - parameters.op                      => op
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
   - name: arw.vserver.state
     matches:
       - name: op
         value: dry-run
     exports:
-      - parameters.op                   => op
-      - parameters.vserverName          => svm
-      - parameters.vserverUuid          => svm_uuid
+      - parameters.op                      => op
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
 
   - name: callhome.arw.activity.seen
     exports:
-      - parameters.volUuid              => volume_uuid
-      - parameters.volName              => volume
+      - parameters.volUuid                 => volume_uuid
+      - parameters.volName                 => volume
+      - parameters.vserverName             => svm
+      - parameters.vserverUuid             => svm_uuid
 
   - name: callhome.battery.low
```

Comment:
we need to add all bookend events' resolving ems

nvram.battery.charging.normal